src/pages/contact_info.tsx

```diff
@@ -0,0 +1,267 @@
+    <div>
+      <Image
+        src="/assets/images/contact_info/Rectangle 1.png"
+        alt="Rectangle 1"
+        height={160}
+        p={25}
+        mb=""
+      />
+    </div>
+
+    <Group justify="space-between" mb="xs">
+      <Text fw={500}>{title}</Text>
+    </Group>
+
+    <Text size="sm" c="dimmed">
+      Operation Hours:
+    </Text>
+
+    <Text>{operationHours}</Text>
+
+    <Button variant="filled" color="blue" fullWidth mt="lg" radius="md" size="lg">
+      {buttonText}
+    </Button>
+  </Card>
+);
+
+
+
+
+// Export
+export default function contactInfoPage() {
+  return <div className="container mx-auto mt-12">
+    <section className="relative w-full">
+      {/* Header */}
+      <h1>
+      <div className="flex flex-col items-center">
+        <div className="text-5xl font-bold mb-4" >Contact Information</div>
+      </div>
+        </h1>
+
+        {/* Header.Description */}
+        <Text align="center" className="mb-16">
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+</Text>
+
+ {/* Cards */}
+ <Grid align='center' style={{ margin: '50px 100px 50px 150px' }}>
+  {/* 1st Card */}
+        <Grid.Col span={4}>
+          <CustomCard
+            title="Academic Support"
+            operationHours="Mon-Fri 08:30 - 17.30"
+            buttonText="Email"
+          />
+        </Grid.Col>
+        <Grid.Col span={4}>
+        {/* 2nd Card */}
+        <CustomCard
+            title="Registra"
+            operationHours="Mon-Fri 08:30 - 17.30"
+            buttonText="Email"
+          />
+          </Grid.Col>
+          <Grid.Col span={4}>
+        {/* 3rd Card */}
+        <CustomCard
+            title="Library"
+            operationHours="Mon-Fri 08:30 - 17.30"
+            buttonText="Email"
+          />
+          </Grid.Col>
+          <Grid.Col span={4}>
+        {/* 4th Card */}
+        <CustomCard
+            title="Finance"
+            operationHours="Mon-Fri 08:30 - 17.30"
+            buttonText="Email"
+          />
+          </Grid.Col>
+          <Grid.Col span={4}>
+        {/* 5th Card */}
+        <CustomCard
+            title="IT Support"
+            operationHours="Mon-Fri 08:30 - 17.30"
+            buttonText="Email"
+          />
+          </Grid.Col>
+          <Grid.Col span={4}>
+        {/* 6th Card */}
+        <CustomCard
+            title="Student Service"
+            operationHours="Mon-Fri 08:30 - 17.30"
+            buttonText="Email"
+          />
+          </Grid.Col>
+      </Grid>
+
+
+
+
+
+      
+
+    </section>
+  </div>}
+
+
+
+
+
+// Backup
+
+/*
```

Comment:
i guess just don't add this ? you can just save it in your computer instead of having it in here.